packages/fields-document/src/DocumentEditor/component-blocks/api.tsx

```diff
@@ -104,7 +104,7 @@ export type ArrayField<ElementField extends ComponentSchema> = {
 
 export type RelationshipField<Many extends boolean> = {
   kind: 'relationship';
-  listKey: string;
+  modelKey: string;
```

Comment:
I think this should probably stay `listKey`

Reply:
This should probably be `listKey` but the distinction only makes sense when singletons exist so was going to do some review on these things in that PR. Want me to fix here first though?

Reply:
Yeah, if it's not too annoying, fixing it here would probably be good since it looks like `listKey` wasn't entirely replaced with `modelKey` anyway